actions/send_websocket_MOD.js

```diff
@@ -62,20 +73,34 @@ module.exports = {
     const WebSocket = Mods.require('ws');
 
     // Eval Message Fields
-    const IP = this.evalMessage(data.IP, cache);
-    const Port = this.evalMessage(data.Port, cache);
-    const Message = this.evalMessage(data.Message, cache);
+    const ip = this.evalMessage(data.ip, cache);
+    const port = this.evalMessage(data.port, cache);
+    const message = this.evalMessage(data.message, cache);
 
-    // Create Connection
-    const ws = new WebSocket(`ws://${IP}:${Port}/`);
-
-    // Catch Error
-    ws.on('error', console.error);
+    const protocol = parseInt(data.protocol, 10);
 
-    // Send Message
-    ws.on('open', function open() {
-      ws.send(`${Message}`);
-    });
+    // Create Connection
+    if (protocol === 0) {
```

Comment:
Remove the if statement
```js
const ws = new WebSocket(`${protocol ? 'wss' : 'ws'}://${ip}:${port}/`);
```

Reply:
Removed if statement and added Lambda code